mavenproject/src/main/java/mtm/Game.java

```diff
@@ -0,0 +1,32 @@
+package mtm;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Game {
+	DeckBuilder deckBuilder = new DeckBuilder();
+	
+	Player player1 = new Player(1);
+	Player player2 = new Player(2);
+	
+	public boolean loadDecks() {
+		
+		boolean player1LoadDeckResult;
+		boolean player2LoadDeckResult;
+		
+		player1LoadDeckResult = deckBuilder.loadCards(player1);
+		player2LoadDeckResult = deckBuilder.loadCards(player2);
+		
+		if(player1LoadDeckResult == false || player2LoadDeckResult == false)

```

Comment:
Checking if something is equal to `false` is pointless, just use `!player1LoadDeckResult`. You could even do:
```
boolean loadDecksSuccessful = deckBuilder.loadCards(player1) && deckBuilder.loadCards(player2)

if (!loadDeckSuccessful)
    return false;
    
return true;
```
A side effect would be that you wouldn't try to load the player2's cards if loading the player1 cards is unsuccessful, because the `&&` would short-circuit. I guess it's likely that if one fails the other one will too.